source/instructors/admin/organization/student-email.rst

```diff
@@ -0,0 +1,32 @@
+.. meta::
+   :description: Allow teachers to see student email addresses
+
+
+.. _student-email:
+
+Show Student Emails
+===================
+The **Show Student Emails** feature can be enabled to show students email addresses in courses
+
+Follow these steps to show student email:
+
+1. Click your profile icon in the lower left corner of the screen.
+
+   .. image:: /img/class_administration/profilepic.png
+      :alt: Profile
+
+2. In the **Organizations** area, click the name of your organization.
+
+   .. image:: /img/class_administration/addteachers/myschoolorg.png
+      :alt: My Organizations
+
+3. Click the **Overview** tab.
+
+   .. image:: /img/manage_organization/orgsettingstab.png
+      :alt: Organization Settings
+
+4. In the **Education Settingse** area, toggle the **Show Student Emails** to enable it. When enabled, teachers in your organization will be able to see the students email addresses
```

Comment:
When enabled, teachers in your organization will be able to see student email addresses.